hazelcast-integration/filter-based-session-replication/src/main/webapp/WEB-INF/hazelcast.xml

```diff
@@ -1,7 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <hazelcast xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://www.hazelcast.com/schema/config
-                               http://www.hazelcast.com/schema/config/hazelcast-config-3.11.xsd"
+                               http://www.hazelcast.com/schema/config/hazelcast-config-4.0.xsd"
```

Comment:
Don't know how to verify the code sample but it depends on HZ 4.1 so upgraded XML to 4.0.